packages/esm-patient-appointments-app/src/appointments/appointments-form.component.tsx

```diff
@@ -52,8 +52,6 @@ const AppointmentsForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, closeW
   }
 
   const handleSubmit = () => {
-    if (!selectedServiceType) return;
```

Comment:
Isn't `selectedServiceType` necessary for submitting?

Reply:
It is not a required field by the backend but we were requiring it since we were getting the duration to make the `endTime` from the serviceType, but now that we mostly have no service types and we can access the durations on the service top level, it's better we don't make it a condition for submitting.

Reply:
@hadijahkyampeire ok, makes sense. What do you think @CynthiaKamau ?

Reply:
@ZacButko do you know if it is possible to make appointment endpoints point to this backend because it is the one that works? `backend=[https://](http://loc/)data.kenyahmis.org:8500 --api-url=/openmrs`

Reply:
@hadijahkyampeire  I'm not sure what you're asking.. for dev3 It would be preferable to get appointments working on that setup. If it's not pointing to a working instance right now we should make a talk post to get backend team to look at it.

if the question is just about syntax the better syntax would be `--backend "https://data.kenyahmis.org --port 8500 ...`